elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -377,5 +391,22 @@ $pfe-cta--Color--fallback: #003366;
     --pfe-cta--Color--hover:            #{pfe-color(ui-complement)};
 }
 
+/// ===========================================================================
+/// WIND CTA VARIANTS
+/// ===========================================================================
 
-
+:host([pfe-priority="secondary"][pfe-variant="wind"]) {
+  --pfe-cta--BackgroundColor:           transparent;
```

Comment:
since the background is transparent, shouldn't all these be listening for broadcast variables first?

Reply:
They should!  I'll update.

Reply:
Done!